src/Toolkit/composer.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "symfony/ux-toolkit",
+  "type": "symfony-bundle",
+  "description": "Twig toolkit for Symfony",
+  "keywords": [
+    "symfony-ux",
+    "twig",
+    "components"
+  ],
+  "homepage": "https://symfony.com",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Symfony Community",
+      "homepage": "https://symfony.com/contributors"
+    },
+    {
+      "name": "Hugo Alliaume",
+      "email": "hugo@alliau.me"
+    },
+    {
+      "name": "Jean-François Lépine",
+      "email": "lepinejeanfrancois@gmail.com"
+    },
+    {
+      "name": "Simon André",
+      "email": "smn.andre@gmail.com"
+    }
+  ],
+  "require": {
+    "php": ">=8.2",
+    "tales-from-a-dev/twig-tailwind-extra": "^0.3.0",
+    "twig/extra-bundle": "^2.12|^3.0",
+    "twig/html-extra": "^3.19",
+    "twig/twig": "^2.12|^3.0",
+    "symfony/console": "^7.2",
+    "symfony/framework-bundle": "^6.4|^7.0",
+    "symfony/phpunit-bridge": "^6.4|^7.0",
+    "symfony/twig-bundle": "^6.4|^7.0",
+    "symfony/ux-twig-component": "^2.22"
+  },
+  "autoload": {
+    "psr-4": {
+      "Symfony\\Ux\\Toolkit\\": "src"
```

Comment:
This is the most "correct" PER naming... but we currently use "UX" (we'll discuss this another time, nothing urgent here)